packages/labs/webpack.config.js

```diff
@@ -8,15 +8,15 @@ const path = require("path");
 
 module.exports = Object.assign({}, baseConfig, {
     entry: {
-        "labs.bundle": [
+        labs: [
```

Comment:
does this change the name of any emitted .css files?

Reply:
good shout -- it does not because extract text works based on the name of the `.scss` file, which we name the same as the output css file by convention. this is also verified by `assert-package-layout` and I checked it just now locally.